cypress/integration/logistics/materialentnahme_partial_HU.js

```diff
@@ -0,0 +1,244 @@
+let packingInstructionsVersionForTU;
+let packingInstructionsVersionForLU;
+
+it('Read the fixture', function() {
+  cy.fixture('logistics/materialentnahme_partial_HU.json').then(f => {
+    productForPackingMaterialTU = appendHumanReadableNow(f['productForPackingMaterialTU']);
+    productForPackingMaterialLU = appendHumanReadableNow(f['productForPackingMaterialLU']);
+    packingMaterialForTU = appendHumanReadableNow(f['packingMaterialForTU']);
+    packingMaterialForLU = appendHumanReadableNow(f['packingMaterialForLU']);
+    packingInstructionsNameForTU = appendHumanReadableNow(f['packingInstructionsNameForTU']);
+    packingInstructionsNameForLU = appendHumanReadableNow(f['packingInstructionsNameForLU']);
+    productName1 = appendHumanReadableNow(f['productName1']);
+    productCategoryName = appendHumanReadableNow(f['productCategoryName']);
+    discountSchemaName = appendHumanReadableNow(f['discountSchemaName']);
+    priceSystemName = appendHumanReadableNow(f['priceSystemName']);
+    priceListName = appendHumanReadableNow(f['']);
+    priceListVersionName = appendHumanReadableNow(f['priceListVersionName']);
+    productType = f['productType'];
+    vendorName = appendHumanReadableNow(f['vendorName']);
+    materialentnahme = f['materialentnahme'];
+    warehouseName = f['warehouseName'];
+    packingInstructionsVersionForTU = appendHumanReadableNow(f['packingInstructionsVersionForTU']);
+    packingInstructionsVersionForLU = appendHumanReadableNow(f['packingInstructionsVersionForLU']);
+  });
+});
+
+describe('Partial material withdrawal in handling unit editor with Materialentnahmelager', function() {
+  it('Create price entities', function() {
+    Builder.createBasicPriceEntities(priceSystemName, priceListVersionName, priceListName, false);
+    cy.fixture('discount/discountschema.json').then(discountSchemaJson => {
+      Object.assign(new DiscountSchema(), discountSchemaJson)
+        .setName(discountSchemaName)
+        .apply();
+    });
+  });
+  it('create product from 24_Gebinde category in order to use it for packing material for transport unit', function() {
+    // eslint-disable-next-line
+    Builder.createProductWithPriceUsingExistingCategory(priceListName, productForPackingMaterialTU, productForPackingMaterialTU, productType, "24_Gebinde");
+  });
+  it('create product from 24_Gebinde category in order to use it for packing material for loading unit', function() {
+    Builder.createProductWithPriceUsingExistingCategory(
+      priceListName,
+      productForPackingMaterialLU,
+      productForPackingMaterialLU,
+      productType,
+      '24_Gebinde'
+    );
+  });
+  it('create packing material for handling unit - Transport unit', function() {
+    cy.fixture('product/packing_material.json').then(packingMaterialJson => {
+      Object.assign(new PackingMaterial(), packingMaterialJson)
+        .setName(packingMaterialForTU)
+        .setProduct(productForPackingMaterialTU)
+        .setLength('0')
```

Comment:
Would it make sense to add `length`, `width` and `height` to the fixture with value 0 and not override them into the tests? 

If there is a value already set for those in the fixture then ignore my comment here.

Reply:
done.